public/js/utils.js

```diff
@@ -26,11 +25,25 @@ var transformProjects = function (data){
     projObj.destination_type= '',
     projObj.destination_url= '',
     projObj.tool_site_id= item.entityId;
+    projObj.type= item.type;
+    projObj.type_code = getTypeCode(item.type);
     projectsColl.push(projObj);
   });
-  data.data = projectsColl;
-  return data;
+    return projectsColl;
+  //return data;
 }
+
+var getTypeCode = function(type){
+  if (type==='myworkspace'){
+    return 1;
+  }
+  else if (type==='GradToolsStudent'){
```

Comment:
We won't have site typed 'GradToolsStudent'. Only GradToolsRackham and GradToolsDepartment